DQM/L1TMonitor/python/L1TStage2EMTF_cfi.py

```diff
@@ -4,6 +4,8 @@
     "L1TStage2EMTF",
     emtfSource = cms.InputTag("emtfStage2Digis"),
```

Comment:
Use different InputTags for the different collections you are going to use in the module to avoid ProductNotFound exceptions